test/cpp/api/functional.cpp

```diff
@@ -243,6 +243,23 @@ TEST_F(FunctionalTest, ELU) {
   }
 }
 
+TEST_F(FunctionalTest, SELU) {
+  const double scale = 1.0507009873554804934193349852946;
+  const double alpha = 1.6732632423543772848170429916717;
+  for (const auto inplace : {false, true}) {
+    auto input = torch::randn({5, 5});
+    auto expected = scale *
+        (torch::max(torch::zeros_like(input), input) +
+         torch::min(torch::zeros_like(input), alpha * (torch::exp(input) - 1)));
+    auto output = F::selu(input, SELUOptions(inplace));
```

Comment:
nit: we might be able to write
```suggestion
    auto output = F::selu(input, inplace);
```
so that we can test the implicit conversion from `bool` to `SELUOptions` :D